packages/headless/src/features/search-parameters/search-parameter-serializer.ts

```diff
@@ -30,10 +39,16 @@ function serializePair(pair: [string, unknown]) {
     return isFacetObject(val) ? serializeFacets(key, val) : '';
   }
 
-  if (key === 'nf' || key === 'df') {
+  if (key === 'nf') {
     return isRangeFacetObject(val) ? serializeRangeFacets(key, val) : '';
```

Comment:
Let's rename `serializeRangeFacets` to `serializeNumericFacets` since it is no longer managing the date facets,

Reply:
`serializeDateFacets` reuses `serializeRangeFacets`